package.json

```diff
@@ -4,9 +4,10 @@
   "description": "TypeScript loader for webpack",
   "main": "index.js",
   "scripts": {
-    "pretest": "node build",
+    "build": "node scripts/build",
+    "pretest": "npm run build",
     "test": "node ./node_modules/mocha/bin/mocha --reporter spec test/run.js",
```

Comment:
Unrelated, but thoughts on code coverage? Also, this line could be just `mocha --reporter spec test/run.js`